src/schema/metadata/HeadCoilCoordinateSystem.yaml

```diff
@@ -0,0 +1,14 @@
+---
+name: HeadCoilCoordinateSystem
+description: |
+  Defines the coordinate system for the head coils.
+  See
+  [Appendix VIII](/99-appendices/08-coordinate-systems.html)
+  for a list of restricted keywords for coordinate systems.
+  If `Other`, provide definition of the coordinate system in
```

Comment:
I think in some previous PR, that `Other` should instead be `"Other"`, because that is what you would write in the JSON file. Right, @Remi-Gau ?

Reply:
this would also be an issue for other fields, such as the text in `DigitizedHeadPointsCoordinateSystem`

Reply:
I've tried to update the definitions as those kinds of changes have been made, but it's difficult to catch everything. Thank you for noticing this. I will fix this one and `DigitizedHeadPointsCoordinateSystem`.

Reply:
Should be fixed in f3fd1b5.